app/views/api/v1/conversations/partials/_conversation.json.jbuilder

```diff
@@ -39,3 +39,4 @@ json.snoozed_until conversation.snoozed_until
 json.status conversation.status
 json.timestamp conversation.last_activity_at.to_i
 json.unread_count conversation.unread_incoming_messages.count
+json.last_non_activity_message conversation.messages.where.not(message_type: :activity).reorder('id desc').limit(1).first.try(:push_event_data)
```

Comment:
Can we have a `:scope` defined for this in `Message` model?

`: last_non_activity_message => where.not(message_type: :activity).reorder('id desc').limit(1)` or may be without limit.

Reply:
Sure, I can do that.